### PR TITLE
feat: config nodes and oauth2

### PR DESCRIPTION
**Pull request by cowan-macady**
Add config nodes methods (removed from coverage until tests are pushed in next PR)

Change packages and add renovate

https://app.zenhub.com/workspaces/jarvis-backlog-633d858c2508b30020b1c7df/issues/indykite/jarvis-backlog/3071
https://app.zenhub.com/workspaces/jarvis-backlog-633d858c2508b30020b1c7df/issues/indykite/jarvis-backlog/3072
https://app.zenhub.com/workspaces/jarvis-backlog-633d858c2508b30020b1c7df/issues/indykite/jarvis-backlog/3073

### CODE REVIEW

File: jarvis_sdk/model/email.py [hunk 1]
```diff
@@ -0,0 +1,17 @@
...
+        return Email(message.address, message.name)
+
+    def __init__(self, address, name):
+        self.address = address
+        self.name = name
+
+    def __str__(self):
+        return (
+            "Address: " + self.address + "\n"
+            "Name: " + self.name
```
Comment (Septimus4) [inline jarvis_sdk/model/email.py:16]: String interpolation

File: .coveragerc [hunk 2]
```diff
@@ -9,6 +9,21 @@ omit =
...
+  jarvis_sdk/model/email_service_config.py
+  jarvis_sdk/model/email_template.py
+  jarvis_sdk/model/ingest_mapping_config.py
+  jarvis_sdk/model/oauth2_application.py
+  jarvis_sdk/model/oauth2_application_config.py
+  jarvis_sdk/model/oauth2_client_config.py
+  jarvis_sdk/model/oauth2_provider.py
+  jarvis_sdk/model/oauth2_provider_config.py
+  jarvis_sdk/model/auth_flow_config.py
+  jarvis_sdk/model/config_node.py
```
Comment (Septimus4) [inline .coveragerc:25]: Why do we omit all this files ?

**Review by Septimus4 — CHANGES_REQUESTED** [review]

File: .coveragerc (hunk 2 above)
Comment (cowan-macady) [reply .coveragerc:25]: because they are used only in the examples for the moment (not in the methods themselves which use the protobuf config now)

**Review by Septimus4 — APPROVED** [review]

File: jarvis_sdk/model/email_attachment.py [hunk 3]
```diff
@@ -0,0 +1,25 @@
...
+        self.file_name = file_name,
+        self.content = content
+
+    def __str__(self):
+        return (
+            "Content type: " + self.content_type + "\n"
+            "Content id: " + self.content_id + "\n"
+            "Inline: " + self.inline + "\n"
+            "FileName: " + self.file_name + "\n"
+            "Content: " + self.content + "\n"
```
[the remark this answers is not in the document]
Comment (cowan-macady) [reply jarvis_sdk/model/email_attachment.py]: sorry to have forgotten this in the merge

File: jarvis_sdk/model/email.py (hunk 1 above)
Comment (cowan-macady) [reply jarvis_sdk/model/email.py]: sorry to have forgotten this in the merge

**Pull request merged**